src/mon/MDSMonitor.cc

```diff
@@ -2057,9 +2073,8 @@ void MDSMonitor::maybe_replace_gid(mds_gid_t gid,
     dout(10) << " failing and removing " << gid << " " << info.addr << " mds." << info.rank 
       << "." << info.inc << " " << ceph_mds_state_name(info.state)
       << dendl;
-    mon->clog->info() << "MDS standby '"  << info.name
-                      << "' is not responding, removing it from the set of "
-                      << "standbys";
+    mon->clog->info() << "Standby " << info.human_name() << " is not "
```

Comment:
Rephrased to Standby ... is not responding, dropping it", to lose the redundant "standby".  I like keeping the one at the start so that it's very obvious that we lost a standby rather than something more important.